data/sources/unassigned.json

```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
```

Comment:
not sure what this is yet.

Reply:
maybe vanguard ritual weapon ornament, maybe the seasonal exotic quest from mara, a quick manifest dive on this is confusing